distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -86,6 +86,50 @@
                 <exclude>
                     **/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar
                 </exclude>
+                <!--
+                Excluding all Fabric8 jars since they are not needed or enabled and introduce
+                vulnerability CVE-2016-1906. This suppression should be revisited when DDF-3908
+                (Karaf upgrade) has been completed.
+                -->
+                <exclude>
+                    **/io/fabric8/kubernetes-model/2.0.9/kubernetes-model-2.0.9.jar
```

Comment:
Since we are excluding all the fabric8 stuff could we just have one exclude for `**/io/fabric8/**/*`

Reply:
I don't think we are excluding all the fabric8 stuff, just the kubernetes related libraries